terra-jupyter-gatk-ovtf/Dockerfile

```diff
@@ -29,28 +29,40 @@ RUN apt-get update && apt-get install -yq --no-install-recommends \
   && rm -rf /var/lib/apt/lists/* 
 
 # Install GATK
-ENV GATK_VERSION 4.2.4.0
+ENV GATK_VERSION 4.2.4.1
 ENV GATK_ZIP_PATH /tmp/gatk-${GATK_VERSION}.zip
 
 RUN curl -L -o $GATK_ZIP_PATH https://github.com/broadinstitute/gatk/releases/download/$GATK_VERSION/gatk-$GATK_VERSION.zip \
  && unzip -o $GATK_ZIP_PATH -d /etc/ \
  && ln -s /etc/gatk-$GATK_VERSION/gatk /bin/gatk
 
+# Install OpenVINO™ integration with TensorFlow (OVTF)
+ENV TF_VER 2.7.0
+ENV KERAS_VER 2.7.0
+ENV OVTF_VER 1.1.0
+ENV TF_ENABLE_ONEDNN_OPTS 1
+
 ENV PIP_USER=false
 
-# Force reinstalling TF-2.5.1, Keras-2.5.0 to satisfy OVTF compatibility.
-RUN pip install --force-reinstall tensorflow==2.5.1 keras==2.5.0rc0
+# Force reinstalling TF and Keras to satisfy OVTF compatibility.
+RUN pip install --force-reinstall tensorflow==$TF_VER keras==$KERAS_VER
```

Comment:
did you check if forcing a new install of TF breaks GPU support?

Reply:
We added a note on GPU support in the [README](https://github.com/ravi9/terra-docker/tree/ovtf-update-1.1/terra-jupyter-gatk-ovtf#note-on-cuda-enabled-gpu-devices-support)

Please verify like below.
import tensorflow as tf
# CPU and GPU support available. openvino_tensorflow is disabled
tf.config.list_physical_devices()